README.md

```diff
@@ -278,19 +278,20 @@ configuration options apply to all other scopes as well.
 
 By default the resource manager ships with some reasonable scaling limits and
 makes a reasonable guess at how much system memory you want to dedicate to the
-go-libp2p process. For the default definitions see `DefaultLimits` and
-`ScalingLimitConfig.AutoScale()`.
+go-libp2p process. For the default definitions see [`DefaultLimits` and
+`ScalingLimitConfig.AutoScale()`](./limit_defaults.go).
 
 ### Tweaking Defaults
 
-If the defaults seem mostly okay, but you want to adjust one facet you can do
+If the defaults seem mostly okay, but you want to adjust one facet you can
 simply copy the default struct object and update the field you want to change. You can
 apply changes to a `BaseLimit`, `BaseLimitIncrease`, and `LimitConfig` with
 `.Apply`.
 
 Example
 ```
 // An example on how to tweak the default limits
+// Is this correct?
 tweakedDefaults := DefaultLimits
 tweakedDefaults.ProtocolBaseLimit.Apply(BaseLimit{
```

Comment:
I want to confirm this is right.

I'm probably misunderstanding the go but looking at https://github.com/libp2p/go-libp2p-resource-manager/blob/master/limit.go#L102 I would have thought the limits don't apply since "l" has non-zero values.

Reply:
Yes, you're right.